main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
```

Comment:
@vieiralucas good to know!

@mateusduboli great point. Maybe we should start splitting things around, main is getting too big.